java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -573,33 +573,64 @@ public interface UserDefinedInstanceProperty extends InstanceProperty {
-            .defaultValue("m5.4xlarge")
-            .propertyGroup(InstancePropertyGroup.BULK_IMPORT).build();
-    UserDefinedInstanceProperty DEFAULT_BULK_IMPORT_EMR_INITIAL_NUMBER_OF_EXECUTORS = Index.propertyBuilder("sleeper.default.bulk.import.emr.executor.initial.instances")
-            .description("(Non-persistent EMR mode only) The default initial number of EC2 instances to be used as executors in the EMR cluster. " +
-                    "This default can be overridden by a table property or by a property in the bulk import job specification.")
+    UserDefinedInstanceProperty DEFAULT_BULK_IMPORT_EMR_INITIAL_EXECUTOR_CAPACITY = Index.propertyBuilder("sleeper.default.bulk.import.emr.executor.initial.instances")
+            .description("(Non-persistent EMR mode only) The default initial number of capacity units to provision as EC2 " +
+                    "instances for executors in the EMR cluster.\n" +
+                    "This is measured in instance fleet capacity units. These are declared alongside the requested " +
+                    "instance types, as each type will count for a certain number of units. By default the units are " +
+                    "the number of instances.\n" +
+                    "This property is a default which can be overridden by a table property or by a property in the " +
+                    "bulk import job specification.")
             .defaultValue("2")
             .propertyGroup(InstancePropertyGroup.BULK_IMPORT).build();
-    UserDefinedInstanceProperty DEFAULT_BULK_IMPORT_EMR_MAX_NUMBER_OF_EXECUTORS = Index.propertyBuilder("sleeper.default.bulk.import.emr.executor.max.instances")
-            .description("(Non-persistent EMR mode only) The default maximum number of EC2 instances to be used as executors in the EMR cluster. " +
-                    "This default can be overridden by a table property or by a property in the bulk import job specification.")
+    UserDefinedInstanceProperty DEFAULT_BULK_IMPORT_EMR_MAX_EXECUTOR_CAPACITY = Index.propertyBuilder("sleeper.default.bulk.import.emr.executor.max.instances")
+            .description("(Non-persistent EMR mode only) The default maximum number of capacity units to provision as EC2 " +
+                    "instances for executors in the EMR cluster.\n" +
+                    "This is measured in instance fleet capacity units. These are declared alongside the requested " +
+                    "instance types, as each type will count for a certain number of units. By default the units are " +
+                    "the number of instances.\n" +
+                    "This property is a default which can be overridden by a table property or by a property in the " +
+                    "bulk import job specification.")
             .defaultValue("10")
             .propertyGroup(InstancePropertyGroup.BULK_IMPORT).build();
```

Comment:
Might be worth adding validation predicates to these properties just to be safe (isPositiveInteger)